lib/hanami/extensions.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+require_relative "./extensions/action" if Hanami.bundled?("hanami-controller")
```

Comment:
Rubocop will probably complain but given the code that we have below, this structure would make this file easier to read, IMO:
 
```suggestion
if Hanami.bundled?("hanami-controller")
  require_relative "./extensions/action"
end
```